### PR TITLE
 Article page

### PR DESCRIPTION
**Pull request by Drapegnik**

### CODE REVIEW

**Comment by Drapegnik** [comment]
@darjanska you should write styles in `.scss` files, not in `.css`, please fix it

File: src/article-page/article.html [hunk 1]
```diff
@@ -1 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<title>ARTICLE NAME</title>
+    <link rel="stylesheet" href="article.css">
+	<link rel="stylesheet" href="../../node_modules/font-awesome/css/font-awesome.min.css">
+	<link rel="stylesheet" href="../../node_modules/bulma/css/bulma.css">
```
Comment (Drapegnik) [inline src/article-page/article.html]: you should include `bundle.css` and `assets.min.css`, check #5 for more info

**Comment by Drapegnik** [comment]
[articles page markup](https://trello.com/c/gSHrkE3r/14-articles-page-markup)

File: .htmlhintrc [hunk 2]
```diff
@@ -5,7 +5,6 @@
   "attr-no-duplication": true,
   "doctype-first": false,
   "tag-pair": true,
-  "tag-self-close": true,
```
Comment (Drapegnik) [inline .htmlhintrc]: Why you switch off this rule?

File: package.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
     "gulp-autoprefixer": "^4.0.0",
     "gulp-clean-css": "^3.5.0",
     "gulp-concat-css": "^2.3.0",
+    "gulp-connect": "^5.0.0",
```
Comment (Drapegnik) [inline package.json]: Are you use this package?

File: src/article-page/article-page.ejs [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<% var title = 'Login Page' %>
+<% include ../head/head.ejs %>
+<body>
+
+<% include ../navbar/navbarTyndria.ejs %>
```
Comment (Drapegnik) [inline src/article-page/article-page.ejs]: I think it'll be better to have 1 version of any component on every moment of time. Can you merge two navbars on you choice?

File: package.json (hunk 3 above)
Comment (Drapegnik) [inline package.json]: or I forgot to add it?

Comment (tyndria) [inline package.json]: it's used in gulpfile

File: src/article-page/article-page.ejs (hunk 4 above)
Comment (tyndria) [inline src/article-page/article-page.ejs]: yes, sure

File: .htmlhintrc (hunk 2 above)
Comment (tyndria) [inline .htmlhintrc]: do we really need it? I think, it's redundant...

Comment (Drapegnik) [inline .htmlhintrc]: React required all tags with empty body to be self-closing

Comment (tyndria) [inline .htmlhintrc]: O, ok, I will fix it

File: .htmlhintrc [hunk 5]
```diff
@@ -1,16 +1,17 @@
 {
```
Comment (tyndria) [inline .htmlhintrc]: Just sort it :)

**Comment by Drapegnik** [comment]
👍 LGTM

**Pull request merged**